### PR TITLE
Fix typos add missing utils

### PR DESCRIPTION
**Pull request by beprajwal**
# What problem does this change will solve?
- [x] Fix typos and wrong file/folder names
- [x] Add missing storage util

### CODE REVIEW

File: client/.eslintrc.json [hunk 1]
```diff
@@ -3,11 +3,7 @@
     "browser": true,
     "es2020": true
   },
-  "extends": [
-    "plugin:react/recommended",
-    "airbnb-base",
-    "prettier"
-  ],
+  "extends": ["plugin:react/recommended", "airbnb-base", "prettier", "eslint-plugin-leapfrog"],
```
Comment (Prabeshpd) [inline client/.eslintrc.json:6]: Add prettier configuration for spacing

File: client/.eslintrc.json [hunk 2]
```diff
@@ -16,19 +12,13 @@
     "ecmaVersion": 11,
     "sourceType": "module"
   },
-  "plugins": [
-    "react",
-    "prettier"
-  ],
+  "plugins": ["react", "prettier"],
```
Comment (Prabeshpd) [inline client/.eslintrc.json:15]: .

File: client/src/actions/users.js [hunk 3]
```diff
@@ -1,6 +1,6 @@
 import { createAction } from 'redux-actions';
 
-import * as userService from '@Services/users';
+import * as userService from '../services/users';
```
Comment (Prabeshpd) [inline client/src/actions/users.js]: avoid relative paths

File: client/src/actions/users.js [hunk 4]
```diff
@@ -24,7 +24,7 @@ export const FETCH_CURRENT_USER_FULFILLED = 'FETCH_CURRENT_USER_FULFILLED';
 
 export const createUser = createAction(CREATE_USER, userService.create);
 export const fetchCurrentUser = createAction(FETCH_CURRENT_USER, userService.fetchSelf);
-export const fetchUsers = createAction(FETCH_USER, userService.fetchAll);
+export const fetchUsers = createAction(FETCH_USERS_FULFILLED, userService.fetchAll);
```
Comment (Prabeshpd) [inline client/src/actions/users.js]: It only defines actions. Fulfillment of promise is a criteria which do not have any concerns with functions call

File: client/src/reducers/users.js [hunk 5]
```diff
@@ -5,7 +5,7 @@ import {
   FETCH_CURRENT_USER_PENDING,
   FETCH_CURRENT_USER_REJECTED,
   FETCH_CURRENT_USER_FULFILLED,
-} from '@Actions/users';
+} from '../actions/users';
```
Comment (Prabeshpd) [inline client/src/reducers/users.js]: .

File: client/src/services/tokens.js [hunk 6]
```diff
@@ -2,16 +2,6 @@ import * as storage from '../utils/storage';
...
- *
- * @param {{accessToken, refresToken}} params
- */
-export function persist({ accessToken, refreshToken }) {
-  setAccessToken(accessToken);
-  setRefreshToken(refreshToken);
-}
-
 /**
  * Get access token from storage.
```
Comment (Prabeshpd) [inline client/src/services/tokens.js:6]: Are you using any conventions for the function arrangement? Functions are moved from up to down

File: client/src/services/tokens.js [hunk 7]
```diff
@@ -49,6 +39,16 @@ export function setRefreshToken(refreshToken) {
   return storage.set(REFRESH_TOKEN, refreshToken);
 }
 
+/**
+ * Persist token to storage.
+ *
```
Comment (Prabeshpd) [inline client/src/services/tokens.js:44]: .

File: client/src/services/users.js [hunk 8]
```diff
@@ -1,5 +1,5 @@
 import config from '../config';
-import http from '@Utils/https';
+import http from '../utils/https';
```
Comment (Prabeshpd) [inline client/src/services/users.js]: .

File: client/src/services/tokens.js (hunk 6 above)
Comment (beprajwal) [reply client/src/services/tokens.js:6]: Yeah the linter shown error for function usage before declaration

**Review by Prabeshpd — APPROVED** [review]

**Pull request merged**